image.yaml

```diff
@@ -36,11 +36,11 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_730_CR4
+                  ref: EAP_730_SP01
           - name: jboss-eap-7-image
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_730_CR4
+                  ref: EAP_730_SP01
```

Comment:
@luck3y tag is missing: https://github.com/jboss-container-images/jboss-eap-7-image/releases/tag/EAP_730_SP01

I've merged https://github.com/jboss-container-images/jboss-eap-7-image/pull/181 so you can tag it now.